specs/agents/span-links.md

```diff
@@ -0,0 +1,36 @@
+## Span Links
+
+A Span or Transaction MAY be linked to zero or more other Spans/Transactions that are causally related.
+
+Specific use-cases for Span Links:
+
+1. When a single transaction represents the batch processing of several messages, the agent SHOULD be able to link back to the traces that have produced the messages.
```

Comment:
As these are examples and not actual specs for how to handle messaging, I would not use the RFC-style `SHOULD`.
Once we have a spec for how to handle span links for message tracing, we should link to that.